source/developers/references/legacy_guide/testing/index.rst

```diff
@@ -12,6 +12,7 @@ file on GitHub.
 .. toctree::
     :maxdepth: 2
 
+    browsers
```

Comment:
@feanil I want to move the `browsers` page into testing, because it feels silly in a one item section at the very beginning of the guide. Should I put in a redirect for this move, for people coming from the old docs site? Does it matter? ref https://github.com/openedx/edx-documentation/pull/2200

Reply:
I don't think it's necessary.